std/experimental/allocator/package.d

```diff
@@ -605,6 +605,8 @@ unittest
     assert(a == [0, 0, 0]);
 }
 
+import std.experimental.allocator.gc_allocator: isGCSafeAllocator;
```

Comment:
> do you want to say importing this package here and creating cyclic dependencies is bad

No, I mean you still can't have a global selective import because it's still broken.